doc/orchestrator.rst

```diff
@@ -165,3 +165,287 @@ A full example of configuring KeyDB to run in SmartSim is shown below
   export REDIS_CONF=/path/to/keydb.conf
 
   # run smartsim workload
+
+Multidb
+=======
+
+To support computationally intensive tasks like scientific simulations,
+SmartSim provides functionality to automate the deployment of multiple
+databases on an HPC cluster. As data volume increases, consider augmenting
+the number of sharded databases or provision multiple standalone databases,
```

Comment:
I'm not quite sure what you mean by a 'standalone' database. In general, I would avoid using this term since we've already defined cluster orchestrator and co-located orchestrator in this documentation.

Reply:
hmmm I thought that colocated and cluster encapsulate the application as well as the database, and I was trying to keep it in terms of just db. I will change it to discuss orchestrators since the section is about mutli orchs! thank u for catching, also is standalone a db term?